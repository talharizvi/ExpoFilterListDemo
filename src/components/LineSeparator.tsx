import React from "react"
import { View, Text, TouchableOpacity, StyleSheet } from 'react-native';
import colors from "../constants/colors";

const LineSeparator: React.FC=()=>{
    return(<View style={styles.viewStyle}>

    </View>) 
}

const styles = StyleSheet.create({
    viewStyle:{
        backgroundColor: colors.secondary,
        height: 1,
        marginTop:18,
        marginRight: 20,
        marginLeft:2
    }
});

export default LineSeparator