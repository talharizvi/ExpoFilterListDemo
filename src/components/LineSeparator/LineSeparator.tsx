import React from "react"
import { View, StyleSheet } from 'react-native';
import colors from "../../constants/colors";

const LineSeparator: React.FC=()=>{
    return(<View testID="line-separator" style={styles.viewStyle}/>) 
}

const styles = StyleSheet.create({
    viewStyle:{
        backgroundColor: colors.secondary,
        height: 1,
        marginTop:18,
        marginRight: 20,
        marginLeft:2
    }
});

export default LineSeparator