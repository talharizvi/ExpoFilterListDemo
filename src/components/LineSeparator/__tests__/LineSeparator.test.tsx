import React from 'react';
import { render, screen } from '@testing-library/react-native';
import LineSeparator from '../LineSeparator';

describe('LineSeparator component', () => {
    
  it('renders correctly', () => {
    render(<LineSeparator/>)
    expect(screen.getByTestId('line-separator')).toBeDefined()
  });

});