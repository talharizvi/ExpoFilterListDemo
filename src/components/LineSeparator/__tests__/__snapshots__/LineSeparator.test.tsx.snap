// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineSeparator component renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#dadee6",
      "height": 1,
      "marginLeft": 2,
      "marginRight": 20,
      "marginTop": 18,
    }
  }
  testID="line-separator"
/>
`;
